### PR TITLE
feat: add adopting and faq

### PR DESCRIPTION
**Pull request by toumorokoshi**
The homepage currently 404s to missing adopting and faq pages.

In addition, try to remove some Google-specific verbage.

### CODE REVIEW

File: pages/general/adopting.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Adopting AEPs for your company
+
+Currently, we encourage adoption of AEPs via having your APIs adhere to the
+guidance served at https://aep.dev.
+
+In the future, we plan to introduce a broader ecosystem of tooling including:
+
+- A forkable repository which allows for internal mirrors and
```
Comment (earth2marsh) [inline pages/general/adopting.md:8]: Doesn't this conflict somewhat with the FAQ explanation that forks are discouraged?

**Review by earth2marsh — APPROVED** [review]
LGTM. One minor nit question about forking that I'll leave to you.

File: pages/general/adopting.md (hunk 1 above)
Comment (rofrankel) [reply pages/general/adopting.md:8]: We already have a forkable repository (every repo on GH is forkable) - so what exactly would we be introducing here?

File: pages/general/faq.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+- aep.dev prioritizes broadly applicable design guidance, while google.aip.dev
+  prioritizes design guidance this best for Google, such as guidance that is
+  backwards-compatible with existing decisions at the company.
+- aep.dev is protocol-agonistic, and considers gRPC and HTTP + JSON as
+  first-class protocols for which examples and other content will be authored.
+- google.aip.dev encourages forking the guidance, while aep.dev focuses on a
+  set of core specification that should not be forked, paired with a method to
+  add organization-specific guidance that does not contradict the
+  specification.
```
Comment (rofrankel) [inline pages/general/faq.md:18]:
```suggestion
  specification.
  
aep.dev also has the advantage of hindsight, which makes it possible in some cases to provide better guidance than google.aip.dev.  (One small example:  renaming `page_size` to `max_page_size` for requests to paginated API methods.)
```

File: pages/general/adopting.md (hunk 1 above)
Comment (toumorokoshi) [reply pages/general/adopting.md:8]: I'll remove for now so we can get the rest of this merged in.

I was trying to say we'll have a model where you can have extensible AEPs without violating the core ones - but I can work on that wording later.

**Pull request merged**